doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -279,6 +252,51 @@ order from most specific to general is: ``payload``, ``pipetask``, and ``site``.
     subsections are pipetask labels where can override/set runtime settings for
     particular pipetasks (currently no Quantum-specific settings).
 
+**site**
+    settings for specific sites can be set here.  Subsections are site names
+    which are matched to ``computeSite``.  The following are examples for 
+    specifying values needed to match jobs to glideins.
+
+.. code-block:: YAML
+   :caption: HTCondor plugin example
+
+   site:
+     acsws02:
+       profile:
+         condor:
+           requirements: "(GLIDEIN_NAME == &quot;test_gname&quot;)"
+           +GLIDEIN_NAME: "test_gname"
```

Comment:
To my understanding, the provided example is a part of the description of the **site** setting. For that reason, I would expect the code block be indented as the description text is.

The LSST Dev Guide says [[src](https://developer.lsst.io/restructuredtext/style.html#indentation)]:

    ReStructuredText should be indented consistently with the context